src/lib/components/ProjectsPanel/components/Project.module.css

```diff
@@ -0,0 +1,31 @@
+.project {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  cursor: pointer;
+  padding: 24px;
+  border-radius: 32px;
+  transition: 0.2s all ease;
+  outline: none;
+}
+
+.project:hover {
+  background-color: rgba(155, 170, 190, 0.2);
+}
+
+.project:focus,
```

Comment:
Let's add focus/focus-visible for items as logo or project 

![image](https://user-images.githubusercontent.com/22644149/215820422-2f669445-e8d8-46ab-841f-763fe0bd9be3.png)

![image](https://user-images.githubusercontent.com/22644149/215820579-f16535b2-2b2e-4fb1-9589-0df43728c1c0.png)

Reply:
added